samples/nrf9160/at_monitor/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _at_monitor_sample:
+
+nRF9160: AT monitor
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The AT monitor  sample demonstrates how to use the :ref:`at_monitor_readme` library.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160ns
+
+.. include:: /includes/spm.txt
+
+Overview
+********
+
+The sample defines two AT monitors, one for network status notifications (``+CEREG``) and one for received signal quality parameters notifications (``+CESQ``) via the :c:func:`AT_MONITOR` macro.
+The sample then subscribes to both notifications and switches the modem to function mode one to register to the network.
```

Comment:
While setting modem to function mode one (CFUN=1) is fine in this sample to illustrate AT monitor behavior, I'm a bit afraid that someone might think that sending AT command would be the preferred way of registering to the network. I'd assume that we prefer applications to use LTE Link Control library to achieve this, right?

Reply:
I don't think we recommend a way to change function mode. If the application doesn't need all the functionality of the `lte_link_controller`, it could just send `AT+CFUN=1` and be done with it.

Reply:
Yep, we don't recommend a way to change function mode but we have other places talking about registering or connecting to LTE network. I think it's a bit confusing as we have these different methods in various samples and user might not know what's our preferred solution.